src/apps/VirtualApp/VirtualClient.js

```diff
@@ -649,13 +650,12 @@ class VirtualClient extends Component {
 
   onMessage = (videoroom, msg, jsep) => {
     const {t} = this.props;
-    Janus.log(' ::: Got a message (publisher) :::');
-    Janus.log(msg);
-    let event = msg['videoroom'];
+    Janus.log(`::: Got a message (publisher) ::: ${JSON.stringify(msg)}`);
```

Comment:
no reason to stringify explicitly. Try `Janus.log('::: Got a message (publisher) :::', msg);`

Reply:
In some cases I want to have all details in Sentry, which is not possible due to the fact that Sentry will truncate after printing one level of details... 

See "[object MessageEvent]"


console | Received message on data channel: [object MessageEvent]
-- | --
logger | console
arguments | ["Received message on data channel:",{"isTrusted":"[object MessageEvent]","type":"message","currentTarget":"[object RTCDataChannel]","target":"[object RTCDataChannel]"}]